PCSharpGen.sln

```diff
@@ -64,6 +66,18 @@ Global
 		{B0E8946C-AD11-4FFB-93AB-4E06D32C9ED9}.Release|x64.Build.0 = Release|Any CPU
 		{B0E8946C-AD11-4FFB-93AB-4E06D32C9ED9}.Release|x86.ActiveCfg = Release|Any CPU
 		{B0E8946C-AD11-4FFB-93AB-4E06D32C9ED9}.Release|x86.Build.0 = Release|Any CPU
+		{18D329B7-B953-4E4A-A5D9-BD5CCDECC83F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Darn it. Did I add all 3 platforms?  Can we remove them? It's easier to do sooner rather than later.

Reply:
I'll remove them.